src/libs/AxelarQueryAPI.ts

```diff
@@ -290,9 +290,11 @@ export class AxelarQueryAPI {
 
     if (isDestinationChainL2) {
       if (!executeData) {
-        throw new Error(
-          `executeData is required to calculate the L1 execution fee for ${destChainId}`
+        console.warn(
+          `Since you did not provide executeData, this API will not accurately calculate the 
+          total required fee as we will not be able to capture the L1 inclusion fee for this L2 chain.`
         );
+        return [l1ExecutionFee, l1ExecutionFeeWithMultiplier];
```

Comment:
Instead of return [0, 0] here, we can still calculate L1 fee without executeData with less accuracy by passing executeData as `0x`.